kalamuna/kalastatic
# KSTAT-408 Bring in kstat-kss-builder

Would like to see two things before we bring this in....

- Pretty basic homepage.md https://github.com/kalamuna/kstat-kss-builder/issues/14
- Have the styleguide not look like a Whitescreen when there is no given homepage.md https://github.com/kalamuna/kstat-kss-builder/issues/13


## TODO

- [x] Set our own homepage.md default if it doesn't exist
- [x] Make the styelguide use the correct styles.

## QA

1. Check out the `feature/robloach/kstat-kss-builder` branch
2. Run `npm install`
3. Run `npm start`
4. In your browser, visit http://localhost:3000/build/styleguide/index.html
5. See the homepage
6. Browse the different sections of the styleguide
